microcosm_postgres/encryption/v2/encryptors.py

```diff
@@ -131,3 +138,14 @@ def decrypt(self, value: bytes) -> str:
 
         context, encryptor = self.encryptor_context
         return encryptor.decrypt(context, value)
+
+    def beacon(self, value: str) -> str:
+        if self.encryptor_context is None:
+            raise self.EncryptorNotBound()
+
+        _, encryptor = self.encryptor_context
+        beacon = encryptor.beacon(value)
+        if beacon is None:
+            raise self.BeaconKeyNotSet()
```

Comment:
Is there a unit test?

Reply:
added explicit unit test for this. Thanks